src/main/scala/com/databricks/labs/overwatch/ApiMeta.scala

```diff
@@ -0,0 +1,98 @@
+package com.databricks.labs.overwatch
+import org.apache.log4j.{Level, Logger}
+
+/**
+ * Configuration for each API.
+ */
+trait ApiMeta {
```

Comment:
let's add an init log function to this trait and ensure it gets called each time a class that extends this trait is instantiated. This will ensure that any instantiation that extends this will log the details of the entire meta defaults and overrides.